pkg/credstores/credstore.go

```diff
@@ -88,7 +95,33 @@ func setupSQLite(toolRegistry, dsn string) (string, []string, error) {
 
 	dbFile = strings.TrimSuffix(dbFile, ".db") + "-credentials.db"
 
-	return toolRegistry + "/credential-stores/sqlite", []string{
+	toolRef, err := resolveToolRef(toolRegistries, "credential-stores/sqlite")
```

Comment:
we do this because we're not guaranteed that the default tool registry is present in `toolRegistries`, so we make a best-effort attempt to find the required `credential-stores` tools in the available registries.